examples/lock-threads.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'Lock Closed Threads'
+
+on:
+  schedule:
+    - cron: '42 5 * * 1' # 5:42 AM UTC every Monday
+  workflow_dispatch:
```

Comment:
I switched to once a week (I don't think this needs to run every day) and I kept the `workflow_dispatch` trigger so that we _can_ run it manually between weeks as needed (or for testing).